.jenkins/pipelines/Azure/Nightly/Packaging.Jenkinsfile

```diff
@@ -16,16 +16,31 @@ def LinuxPackaging(String version, String build_type, String lvi_mitigation = 'N
                 cleanWs()
                 checkout scm
                 def task = """
-                           cmake ${WORKSPACE}                               \
-                             -DCMAKE_BUILD_TYPE=${build_type}               \
-                             -DCMAKE_INSTALL_PREFIX:PATH='/opt/openenclave' \
-                             -DCPACK_GENERATOR=DEB                          \
-                             -DLVI_MITIGATION=${lvi_mitigation}             \
+                           cmake ${WORKSPACE}                                         \
+                             -G Ninja                                                 \
```

Comment:
Could you elaborate on what the failure was an why the switch to ninja fixes it? Also, what is different about the previous ctest run of test/samples from the new test in which does similar things but in script? These are all useful pieces of info to comment the code changes with.